### ISSUE TITLE
zoomable_image completely blocks ctrl+wheel zoom (wheel or trackpad)

### ISSUE THREAD

**Issue by hibiyasleep** [mastodon/mastodon#20954]
### Steps to reproduce the problem

1. bring any macOS with any Magic Trackpad equipped (or any system with mouse)
2. open any image preview
3. do a pinch zoom on it (or ctrl+wheel)

### Expected behaviour

image should be zoomed

### Actual behaviour

no zoom happens, even native zoom actions are blocked

### Detailed description

What happens inside:

1) macOS Trackpad 'Pinch to zoom' creates `WheelEvent` w/ `ctrlKey: true`, **not touch event**
2) firing WheelEvent on zoomimage calls `mouseWheelHandler`
3) `mouseWheelHandler` calls `preventDefault()` on _any_ event (which is, please, think thrice before it)

https://github.com/mastodon/mastodon/blob/76ff45230630c32dd01fd45c89557e092a81b27f/app/javascript/mastodon/features/ui/components/zoomable_image.js#L200-L203

4) since it's `preventDefault()`'d, `normalizeWheel` should be manually handle this anyway, but don't;
    it only does moving, not zooming.

---

When I bring touchscreen on Windows machine and pinch-zoom a image, mouse scroll does the excepted job (vertical scroll), but mouse dragging only works vertically if zoomed image is narrower than screen.

### Specifications

* macOS 12.4, Chrome 107, Magic Trackpad 2 (internal and external)
* Windows 10, Chrmoe 107, normal touchscreen, normal wheel mouse

not reproducible on:
* iPadOS 15.4.x, Safari, Magic Trackpad 2 (via Universial Control)
    in this setup, native (not custom) zoom in is normally working.

**Comment by ianp5a** [comment]
Zoom not possible in web client either on v4.0.2.

1. Click on image in a post.
2. The image is displayed over the client with a close X button
3. Attempt to zoom with CTLRL+ Mouse wheel.
4. See it does not zoom.
Expectation: Image should enlarge right up to the edges. Zoom normally works in most good applications. Useful for small details, small text, and especially for older, long sighted people.

**Comment by elias6** [comment]
This is happening to me, and I think it is annoying.

I don't think I'll be able to investigate this in detail any time soon. But I was playing around with my browser dev tools, and I discovered that if I disable the `pointer-events: auto` CSS rule for the `.modal-root__modal` selector, this no longer happens.

**Comment by frumble** [comment]
Also reproducible with Magic Trackpad on Linux in Firefox (with MOZ_USE_XINPUT2=1 / MOZ_ENABLE_WAYLAND=1, which enables the trackpad gestures).

**Comment by kcarrandale** [comment]
Yes, this is a usability and accessibility issue. I'm not sure at what exact point it started affecting mobile pinch zoom (because it isn't reflected in the thread title), but for me it became an issue some point after v4.1.6.

(I'm in two instances - the one still on v4.1.6 has pinch zoom in Firefox mobile, while the one on v4.2.1 no longer does after updating.)

**Comment by brendanjones** [comment]
Yeah, annoying bug. If you want to zoom in on a pic you currently have to not be viewing the image preview, which is counter intuitive. 

Is this maybe happening because of the `Expand image view box` button existing, which zooms the image an exact amount? As in that locks the possible zoom actions somehow? I'm not a dev, take this suggestion with a grain of salt.
